doc/api/http2.md

```diff
@@ -159,18 +159,16 @@ an `Http2Session`.
 added: v8.4.0
 -->
 
+* `type` {Integer} An integer identifying the frame type.
+* `code` {Integer} An integer identifying the error code.
+* `id` {Integer} An integer identifying the stream (or 0 if the frame is not
```

Comment:
`` 0 `` -> `` `0` ``?

Reply:
`{integer} The stream id (or 0 if the frame is not associated with a stream).